packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -584,6 +584,12 @@ export function makeVatKeeper(
       JSON.stringify({ snapshotID, startPos: endPosition }),
     );
     addToSnapshot(snapshotID);
+    kernelSlog.write({
+      type: 'heap-snapshot-save',
+      vatID,
+      snapshotID,
+      endPosition,
+    });
```

Comment:
I wish we could somehow flow the console data into the slog as that seems like a more reliable debugging place.

Also it would really help XS trace debugging if we could somehow record the temp file name that was used, but that's tricky since a snapshot command may collide with an existing snapshot and get re-used.

Reply:
Console output is virtually useless for post processing, and I actually would like to stop relying on it so much for following status.

This was more a wish. I had looked at the threading of these values and didn't see a non-convoluted way to get there.